src/meta/meta_bulk_load_service.cpp

```diff
@@ -1710,6 +1768,9 @@ void bulk_load_service::do_continue_app_bulk_load(
     const int32_t same_count = pinfo_map.size() - different_count;
     const int32_t invalid_count = partition_count - pinfo_map.size();
 
+    if (!_meta_svc->try_lock_meta_op_status(meta_op_status::type::BULKLOAD)) {
+        derror_f("fatal, the op status of meta server must be meta_op_status::type::FREE");
```

Comment:
I think there should add error handling here, if bulk load can not acquire lock, it still moves on without any handling.